tests/monster_test_generated.h

```diff
@@ -11,36 +11,46 @@ namespace MyGame {
 
 struct InParentNamespace;
 struct InParentNamespaceT;
+bool operator==(const InParentNamespaceT &lhs, const InParentNamespaceT &rhs);
```

Comment:
this is a bit ugly, mixing them with struct pre-declares. Can you give them their own section below?